src/physics/utils/tropopause_climo_read.F90

```diff
@@ -13,6 +13,7 @@ module tropopause_climo_read
 
   implicit none
   private
+  save
```

Comment:
Is this actually needed?  If not then I would vote to remove it.  Alternatively if it is needed it would be good to know what exactly is being deallocated.